src/common/snippets/src/lowered/pass/allocate_buffers.cpp

```diff
@@ -78,7 +78,7 @@ bool AllocateBuffers::run(lowered::LinearIR& linear_ir) {
         pipeline.register_pass<NormalizeBufferIDs>();
         pipeline.run(linear_ir);
     } else {
-        InitBuffersDefault(m_buffer_scratchpad_size).run(linear_ir);
+        InitBuffersDefault(m_buffer_scratchpad_size).run(linear_ir, linear_ir.begin(), linear_ir.end());
```

Comment:
If we pass constant iteratior... 😃 
```suggestion
        InitBuffersDefault(m_buffer_scratchpad_size).run(linear_ir, linear_ir.cbegin(), linear_ir.cend());
```